src/microbe_stage/TweakedProcess.cs

```diff
@@ -16,6 +16,8 @@ public struct TweakedProcess
 
     public float Rate;
 
+    public float SpeedMultiplier = 1;
```

Comment:
Looks like the comment on line 8 is now outdated and should say "packs a few floats ..."